exasol_script_languages_container_tool/lib/api/generate_language_activation.py

```diff
@@ -1,17 +1,24 @@
 import textwrap
 from pathlib import Path
+from typing import Tuple
+
+from exasol_integration_test_docker_environment.lib.api.common import cli_function
 
 from exasol_script_languages_container_tool.lib.tasks.upload.language_definition import LanguageDefinition
 
 
+@cli_function
 def generate_language_activation(
         flavor_path: str,
         bucketfs_name: str,
         bucket_name: str,
         container_name: str,
-        path_in_bucket: str = ''):
+        path_in_bucket: str = '') -> Tuple[str, str, str]:
     """
     Generate the language activation statement.
+    :return: A tuple of language definition statements: The first one is the alter session statement,
```

Comment:
I think, at some point, we should return the alias and url separated for each aliases. Or, an object which stores them and can generate the alter session and system command. With that, we could also change the alias.

Reply:
Created #171